### ISSUE TITLE
allow hooking intro crashreports

### ISSUE THREAD

**Issue by AEnterprise** [NOVA-Team/NOVA-Core#182]
With forge you can hook intro crash reports from your TE and add additional data to the crashreport, for complex stuff this can be very helpfull to trace the cause of the issue or to replicate it

**Comment by Victorious3** [comment]
Sounds too game specific for me. You have the exception, put your information there.

**Comment by AEnterprise** [comment]
not always, take for example a multiblock, and it throws a NPE but that's caused by some situation that could only heapen if the multiblock was build in a specific direction, if i knew how to do this (https://github.com/AEnterprise/Kinetic-Revolution/blob/master/src/main/java/kineticrevolution/multiblocks/tileEntities/TileMultiBlockBase.java#L39-L47) then it would have made tracing it down so much easier

my point is that sometimes you might want to be able to put status info in a crash report to help tracing things down as it's often hard to get the specific information from the user

**Comment by RX14** [comment]
Maybe some kind of debuginfo hook that the mod provides that returns a String[] of stuff to add to the crashreport and maybe F3 menu?

**Comment by AEnterprise** [comment]
for the crashreport it should only be added if the block in question is the one causing the crash

**Comment by RX14** [comment]
Yeah, maybe not show it on the F3 screen, but I believe that this would be possible.
